wikimedia/wikipedia-ios
# Diffs - Add localized strings and icons for diff user button menu actions

**Phabricator:** https://phabricator.wikimedia.org/T335579 (partially)

### Notes
Adds localized strings and icons to the user button menu actions in the updated diff view header.

### Test Steps
1. Open article history and tap a revision.
2. Tap the from and to user buttons in the header.
3. Confirm icons appear for each menu action.
4. Confirm tapping the actions in the menu route the user to the appropriate views in app.
